app/services/db/profile/has_liveness_check.rb

```diff
@@ -0,0 +1,11 @@
+module Db
+  module Profile
+    class HasLivenessCheck
+      def self.call(user_id)
+        components = ::Profile.where(user_id: user_id, active: true).first&.proofing_components
+        return unless components
+        JSON.parse(components)['liveness_check']
```

Comment:
WDYT of making all of this a method on `Profile`? Then we could do `user.active_profile.has_liveness_check?`

```ruby
class Profile
  # ...
  def has_liveness_check?
     return false unless proofing_components.present?
    JSON.parse(proofing_components)['liveness_check']
  end
end

Reply:
Sounds good.  I'll leave the service param as a user_id.

Reply:
No I meant, why does this separate service class exist? In all the contexts I could find in the app, we have access to `current_user` and make queries for `active_profile` already, we can save a few DB queries each time by using the already-cached versions of those

For example:
https://github.com/18F/identity-idp/pull/3804/files#diff-d7911fa9dd9ec853532c3a8d774840d4R55-R57

This already has `current_user` and `current_user.active_profile` loaded